modules/appeals_api/config/locales/en.yml

```diff
@@ -14,4 +14,8 @@ en:
         no_evidence_submission_accepted: "Corresponding Notice of Disagreement 'boardReviewOption' must be 'evidence_submission'"
         not_found: "%{type} with uuid %{id} not found"
         not_homeless_address_missing: "If not homeless, address must be provided: '/data/attributes/veteran/address'"
+        pdf_download_expired: "PDF for %{type} with id %{id} cannot be downloaded because the form data was deleted from the server after the PII retention period expired"
+        pdf_download_not_ready: "PDF for %{type} with id %{id} cannot be downloaded yet because the appeal is still being submitted"
+        pdf_download_in_error: "PDF for %{type} with id %{id} cannot be downloaded because there was an error in the submission process"
         invalid_schema_type: "The '%{schema_type}' request parameter is invalid"
+        unauthorized_pii_headers: "The PII provided in headers was not sufficient to authorize the request"
```

Comment:
I will need to make a follow-up PR to this one so that I can add examples of each response to the rswag docs - it would put me above the PR line limit to add them here